tests/CMakeLists.txt

```diff
@@ -36,14 +36,14 @@ set(FAISS_TEST_SRC
   test_common_ivf_empty_index.cpp
   test_callback.cpp
   test_utils.cpp
+  test_knn_functions.cpp
 )
 
 add_executable(faiss_test ${FAISS_TEST_SRC})
 
 include(../cmake/link_to_faiss_lib.cmake)
 
 link_to_faiss_lib(faiss_test)
-
 target_link_libraries(faiss_test PUBLIC faiss_example_external_module)
```

Comment:
On latest code, this line is deleted

Reply:
this conflict has resolved

Reply:
Sorry if I was not clear: this line should be deleted. The latest code on master deletes it, so we should not re-add it.
(I am still reading through the test, sorry if there is some delay there)

Reply:
oh my god, i maybe midunderstood your meaning, I thought it was a blank line problem.
Sadddddddddd :(